server/util/getUsersFromFile.js

```diff
@@ -1,9 +1,8 @@
-const path = require('path');
-const fs = require('fs/promises');
-
 const getUsersFromFile = async file => {
-    const roleGroupPath = path.join(__dirname, file);
-    const roleGroupTxt = await fs.readFile(roleGroupPath, { encoding: 'utf8' });
+    const response = await fetch(
```

Comment:
do we have any provision/idea what we would want to do if github is unavailable/this request times out?  We probably want to try/catch it somehow with a fallback/specific error?  I know it seems unlikely considering this only gets called from the oauth login workflow coming back from github.  (this might not need any changes, unless we also check the local file, in which case we'd want to fall back to that)

Reply:
Good point. I will discuss this with @howard-e and @alflennik.

Reply:
Hey this is true, but this code runs immediately after completing a round trip to github. So github would have to fail between completing the round trip and the next request, so it's a very short window. Also, we previously used github to load the admin and no issues occurred that I know of.